generator/templates/base_struct_function.h

```diff
@@ -0,0 +1,49 @@
+{% include 'copyright.txt' %}
```

Comment:
```jinja2
{#- To avoid code duplication was crated this parent file, which contain common part used in:
    "templates/functions/template.h" and "templates/structs/template.h". -#}
```
done in https://github.com/smartdevicelink/sdl_ios/pull/1556/commits/a94c956876980df6680ffe78bd3f268351f39442